code/game/objects/items/kitchen.dm

```diff
@@ -55,6 +55,44 @@
 	else
 		return ..()
 
+/obj/item/kitchen/knife/poison
+	name = "venom knife"
+	icon_state = "poisonknife"
+	force = 12
+	throwforce = 15
+	throw_speed = 5
+	throw_range = 7
+	var/amount_per_transfer_from_this = 5
+	var/list/possible_transfer_amounts = list(3,5)
+	desc = "An infamous knife of syndicate design, it has a tiny hole going through the blade to the handle which stores toxins."
+	materials = null
+
+/obj/item/kitchen/knife/poison/Initialize()
+	. = ..()
+	create_reagents(40,OPENCONTAINER)
+
+/obj/item/kitchen/knife/poison/attack_self(mob/user)
+	if(possible_transfer_amounts.len)
+		var/i=0
+		for(var/A in possible_transfer_amounts)
+			i++
+			if(A == amount_per_transfer_from_this)
+				if(i<possible_transfer_amounts.len)
+					amount_per_transfer_from_this = possible_transfer_amounts[i+1]
+				else
+					amount_per_transfer_from_this = possible_transfer_amounts[1]
+				to_chat(user, "<span class='notice'>[src]'s transfer amount is now [amount_per_transfer_from_this] units.</span>")
+				return
+
+/obj/item/kitchen/knife/poison/attack(mob/living/M, mob/user)

```

Comment:
I kind of want to ask for a fuckin can_inject check that reduces the amount injected

Reply:
uhh sure I'll look at syringes for how to do that

Reply:
oh yeah for balance against hardsuits makes sense

Reply:
now it will only inject 1u if the target has inject resistance